Resources/Textures/Objects/Tools/lantern.rsi/meta.json

```diff
@@ -0,0 +1,99 @@
+﻿{
```

Comment:
This RSI is missing copyright and license information

Reply:
Added copyright information.